programs/zstdcli.c

```diff
@@ -644,6 +649,7 @@ int main(int argCount, const char* argv[])
                     if (!strcmp(argument, "--train")) { operation=zom_train; if (outFileName==NULL) outFileName=g_defaultDictName; continue; }
                     if (!strcmp(argument, "--maxdict")) { nextArgumentIsMaxDict=1; lastCommand=1; continue; }  /* kept available for compatibility with old syntax ; will be removed one day */
                     if (!strcmp(argument, "--dictID")) { nextArgumentIsDictID=1; lastCommand=1; continue; }  /* kept available for compatibility with old syntax ; will be removed one day */
+                    if (!strcmp(argument, "--shrink-dict")) { nextArgumentIsShrinkDict=1; lastCommand=1; continue; }  /* kept available for compatibility with old syntax ; will be removed one day */
```

Comment:
Add this to `parseFastCoverParameters` and `parseCoverParameters`. Maybe just call it `shrink`?

Check how `split` is implemented.